tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -1790,11 +1830,11 @@ main (int argc, char **argv)
   testing::InitGoogleTest (&argc, argv);
 
   /* ignore tizen feature status while running the testcases */
-  ml_set_feature_status (1);
+  set_feature_state (1);
```

Comment:
Is that better? What is a reason that you want to remove prefix (e.g. ml_****)?

Reply:
The macro set_feature_state() - check ML feature in Tizen - had no prefix, so I defined new macros without it.